checker/tests/resourceleak-arrays/owningArrayField.java

```diff
@@ -0,0 +1,230 @@
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.calledmethodsonelements.qual.*;
+import org.checkerframework.checker.calledmethodsonelements.qual.EnsuresCalledMethodsOnElements;
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.mustcallonelements.qual.*;
+
+@InheritableMustCall({"close", "foo"})
+class Resource {
+  public void foo() {}
+
+  public void close() {}
+}
+
+class illegalOwningArrayField {
```

Comment:
nit: Java class and file names should always start with a capital letter, by convention.